src/main/webapp/app/shared/form/title-channel-name/title-channel-name.component.ts

```diff
@@ -43,10 +41,9 @@ export class TitleChannelNameComponent implements OnInit {
         this.formatChannelName(this.channelNamePrefix + this.title);
     }
 
-    formatChannelName(newName: string) {
-        if (!this.hideChannelName) {
-            this.channelName = newName.toLowerCase().slice(0, 30).replaceAll(' ', '-');
-            this.channelNameChange.emit(this.channelName);
-        }
+    formatChannelName(newName: string, allowDuplicateDashes = false) {
```

Comment:
when do we want to allow duplicate dashes? What's a use case for this?

Reply:
This is the case, if the user manually enters a channel name. We could restrict it further, but there is no technical reason for it, so I wouldn't do it. We also do not restrict it for standard channels.